subt/artf_node.py

```diff
@@ -27,6 +28,26 @@
 }
 
 
+def check_results(result, result_cv):
+    ret = []
+    if len(result) == 0 or len(result_cv) == 0:
+        return ret
+    for r_cv in result_cv:
+        name_cv, score_cv, bbox = r_cv
+        bbox_x1, bbox_y1, bbox_x2, bbox_y2 = bbox
+        for r in result:
+            name, points = r
+            if name != name_cv:
+                continue
+            x = np.array([p[0] for p in points])
+            y = np.array([p[1] for p in points])
+            x_in_bbox = (x > bbox_x1) & (x < bbox_x2)  # arr of boolean values
+            y_in_bbox = (y > bbox_y1) & (y < bbox_y2)  # arr of boolean values
+            if sum(x_in_bbox & y_in_bbox) >= 1:  #at least one point is in the bbox
```

Comment:
`np.any(np.logical_and(x_in_bbox, y_in_bbox))` would be closer to the idea what the code should do, even if functionally equivalent.

Reply:
Ok, thanks for suggestion.